README.md

```diff
@@ -1,10 +1,10 @@
 # Public Curation Protocol
 
 ### High-level overview
-1. CurationFactory.sol allows for easy creation of individual curation contracts
-2. Individual curation contracts are ERC721 collections themselves, with curators recieving a non-transferable "curationReciept" token that contains the information of the listing they have curated. Curators can "remove" listing by burning their curationReciepts
-3. Active listings on a given curation contract can be retreived by the "getListings()" view call on a given Curator.sol proxy, or be using NFT indexers to gather data on all curationReciepts that have been minted from a given curation contract
-4. Listings contain the data specified in the "Listing" struct found in [https://github.com/public-assembly/curation-protocol/blob/main/src/interfaces/ICurator.sol](ICurator.sol)
+1. Factory allows for easy creation of individual curation contracts
+2. Individual curation contracts are ERC721 collections themselves, with curators recieving a non-transferable "listingToken" that contains the information of the listing they have curated. Curators can "remove" listing by burning their curationReciepts
+3. Active listings on a given curation contract can be retreived by the "getListings()" view call on a given Curator.sol proxy, or by using NFT indexers to gather data on all curationReciepts that have been minted from a given curation contract
```

Comment:
I feel like if you used Active here you have to use inactive while referenced burned tokens too

Reply:
Also hyperlink curator.sol to the repo